spec/services/spotify_authenticator_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe SpotifyAuthenticator do
+  describe "#authenticate" do
+    it "returns access token from Spotify API" do
+      post_response = double(:post_response,
+                             body: "{\"access_token\":\"BQDgqk2108QWdiluKnJ_QvUddeupO2hH4vOXJpfxnE4GSOd4jFIMX20GYXfykcaeBlTZ8Sad4RqEFqAnx1X_6Q\",\"token_type\":\"Bearer\",\"expires_in\":3600}"
```

Comment:
Quoting pro-tip: use `%q` or `%Q` to avoid backslashing quotes:

```ruby
irb(main):002:0> x = 8
=> 8
irb(main):003:0> %q(foo "bar" baz #{x} times)
=> "foo \"bar\" baz \#{x} times"
irb(main):004:0> %Q(foo "bar" baz #{x} times)
=> "foo \"bar\" baz 8 times"
```